shared/common-adapters/section-list.desktop.tsx

```diff
@@ -234,15 +236,12 @@ class SectionList extends React.Component<Props, State> {
       <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.container}>
         {this.props.disableAbsoluteStickyHeader && stickyHeader}
         <Kb.ScrollView
-          style={Styles.collapseStyles([styles.scroll, this.props.style])}
+          style={Styles.collapseStyles([styles.scroll, this.props.contentContainerStyle])}
```

Comment:
did you check this rename didn't break a ton of things

Reply:
Yes. Though, it turns out it broke 1 thing that is fixed in https://github.com/keybase/client/pull/23049